pkg/plugins/manager/pipeline/initialization/steps_test.go

```diff
@@ -28,7 +29,7 @@ func TestInitializer_Initialize(t *testing.T) {
 			Class: plugins.ClassCore,
 		}
 
-		stepFunc := BackendClientInitStep(&fakeEnvVarsProvider{}, &fakeBackendProvider{plugin: p})
+		stepFunc := BackendClientInitStep(&fakeEnvVarsProvider{}, &fakeBackendProvider{plugin: p}, tracing.InitializeTracerForTest())
```

Comment:
It would be good to add our own fake / noop tracer (same for pkg/plugins/backendplugin/coreplugin/core_plugin_test.go). We can add a mock [here](https://github.com/grafana/grafana/blob/main/pkg/plugins/manager/fakes/fakes.go) maybe?